kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -898,6 +927,180 @@ struct ResetMetadata {
   }
 };
 
+struct ListColumnFamilies {
+  const bool read_only = true;
+  std::string description() const {
+    return "listColumnFamilies\n"
+           " List column families in Rocksdb";
+  }
+
+  std::string toJson(const std::unordered_set<std::string> &result) const {
+    std::ostringstream oss;
+    oss << "{\n\t\"column_families\":[";
```

Comment:
Nit: I think we can use two spaces and not `\t` in interest of doing what other commands do.

Reply:
Done